src/vma/dev/cq_mgr_mp.cpp

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2001-2017 Mellanox Technologies, Ltd. All rights reserved.
+ *
+ * This software is available to you under a choice of one of two
+ * licenses.  You may choose to be licensed under the terms of the GNU
+ * General Public License (GPL) Version 2, available from the file
+ * COPYING in the main directory of this source tree, or the
+ * BSD license below:
+ *
+ *     Redistribution and use in source and binary forms, with or
+ *     without modification, are permitted provided that the following
+ *     conditions are met:
+ *
+ *      - Redistributions of source code must retain the above
+ *        copyright notice, this list of conditions and the following
+ *        disclaimer.
+ *
+ *      - Redistributions in binary form must reproduce the above
+ *        copyright notice, this list of conditions and the following
+ *        disclaimer in the documentation and/or other materials
+ *        provided with the distribution.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+ * BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#include "dev/cq_mgr_mp.h"
+#include "dev/cq_mgr_mlx5.inl"
+#include "dev/qp_mgr_mp.h"
+
+
+#define MODULE_NAME 		"cqm"
+
+#define cq_logpanic 		__log_info_panic
+#define cq_logerr		__log_info_err
+#define cq_logwarn		__log_info_warn
+#define cq_loginfo		__log_info_info
+#define cq_logdbg		__log_info_dbg
+#define cq_logfunc		__log_info_func
+#define cq_logfuncall		__log_info_funcall
+#define cq_logfine		__log_info_fine
+
+
+#ifdef HAVE_MP_RQ
+
+const uint32_t cq_mgr_mp::UDP_OK_FLAGS = IBV_EXP_CQ_RX_IP_CSUM_OK |
```

Comment:
It's worth commenting that you're using the const variable to save repeating the logical _or_ operation every time you successfully poll.